src/org/jitsi/impl/neomedia/rtp/translator/OutputDataStreamImpl.java

```diff
@@ -250,17 +250,34 @@ private int doWrite(
                             format,
                             exclusion);
             }
-            if (!write)
-                continue;
 
-            // Allow the RTPTranslatorImpl a final chance to filter out the
-            // packet on a source-destination basis.
-            write
-                = translator.willWrite(
+            if (write)
+            {
+                // Allow the RTPTranslatorImpl a final chance to filter out the
+                // packet on a source-destination basis.
+                write
+                    = translator.willWrite(
                         /* source */ exclusion,
-                        buf, off, len,
+                    buf, off, len,
                         /* destination */ streamRTPManager,
-                        _data);
+                    _data);
+            }
+
+            // Hide gaps in the sequence numbers because of dropping packets.
+            Long ssrc = RawPacket.getSSRCAsLong(buf, off, len);
```

Comment:
This code executes for RTCP as well. I suspect that up until now it somehow worked without major issues, but with this it will quickly fill the map since getSSRCAsLong will actually read the NTP timestamp if the packet is RTCP, and that changes with each packet.